doc/RELEASE_NOTES.md

```diff
@@ -0,0 +1,832 @@
+
+There are Ansible [playbooks](https://galaxy.ansible.com/pbench/agent) available via Ansible Galaxy to install the `pbench-agent`, and the pieces needed (key and configuration files) to be able to send results to a Pbench Server.  To use the RPMs provided above via COPR with the playbooks, your inventory file needs to include the `fedoraproject_username` variable set to `ndokos`, for example:
+
+```
+...
+
+[servers:vars]
+fedoraproject_username = ndokos
+pbench_repo_name = pbench-test
+
+...
+```
+
+Alternatively, one can specify `fedoraproject_username` on the command line, rather than having it specified in the inventory file:
+
+    ansible-playbook -i <inventory> <playbook> -e '{fedoraproject_username: ndokos}' -e '{pbench_repo_name: pbench-test}'
+
+_**NOTE WELL**_: If the inventory file also has a definition for `pbench_repo_url_prefix` (which was standard practice before `fedoraproject_username` was introduced), it needs to be deleted, otherwise it will override the default repo URL and the `fedoraproject_username` change will not take effect.
+
+While we don't include installation instructions for the new `node-exporter` and `dcgm` tools in the published documentation, you can find a manual installation procedure for the Prometheus "node_exporter" and references to the Nvidia "DCGM" documentation in the [`agent/tool-scripts/README`](https://github.com/distributed-system-analysis/pbench/blob/v0.71.0-beta.0/agent/tool-scripts/README.md).
+
+Container images built using the above RPMs are available in the [Pbench](https://quay.io/organization/pbench) organization in the Quay.io container image repository using tags `beta`, `v0.71.0-2`, and `9d90a97cc`.
+
+
+Summary of Changes
+====
+
+## Support for RHEL 9 & CentOS Stream 9
+
+Support for RHEL & CentOS Stream 9 is provided in this release.
+
+
+## The New "Tool Meister" Sub-System
+
+The "Tool Meister" sub-system (introduced by PR #1248) is the major piece of functionality delivered with the release of `v0.71` of the pbench-agent.
+
+This is a significant change, where the pbench-agent first orchestrates the instantiation of a "Tool Meister" process on all hosts registered with tools, using a Redis instance to coordinate their operation, and the new "Tool Data Sink" process handles the collection of data into the pbench run directory hierarchy.  This effectively eliminates all remote SSH operations for individual tools except the initial one per host to create each Tool Meister instance.
+
+One Tool Meister instance is created per registered host, and then a single Tool Data Sink instance is created on the host where the benchmark convenience script is run.  The Tool Meister instances are responsible for running the registered tools on their respective host, collecting the data generated as appropriate. The Tool Data Sink is responsible for collecting and storing locally all data sent to it from the deployed Tool Meister User.
```

Comment:
The last word in line 77 changed from "instances" to "User"...was that intentional?

Reply:
No, I am not sure what I did, but I must have either unintentionally caused Emacs to make some global replacement, or there is a gremlin running around.